examples/distro/build.gradle

```diff
@@ -29,10 +29,8 @@ subprojects {
   }
 
   repositories {
-    maven {
-      url = uri("https://oss.sonatype.org/content/repositories/snapshots")
```

Comment:
Can we start using SNAPSHOTs in distro example too? (Separate PR?)

Reply:
Absolutely, will fix right away

Reply:
Turns out, this cannot be done until we fix publishing of `agent-for-testing`.

Reply:
Done